docker/qgisserver/geomapfish_qgisserver/accesscontrol.py

```diff
@@ -45,14 +44,20 @@ def __init__(self, server_iface):
         super().__init__(server_iface)
 
         self.server_iface = server_iface
+        self.initialized = False
 
         try:
             config.init(os.environ.get("GEOMAPFISH_CONFIG", "/etc/qgisserver/geomapfish.yaml"))
 
-            c2cwsgiutils.broadcast.init(None)
+            c2cwsgiutils.broadcast.init()
 
             configure_mappers()
-            engine = sqlalchemy.create_engine(config.get("sqlalchemy_slave.url"))
+            QgsMessageLog.logMessage(
+                "Connect to the database: {}".format(config.get("sqlalchemy_slave.url")),
+                "GeoMapFishAccessControl",
+                level=Qgis.Info,
+            )
+            engine = sqlalchemy.create_engine(config["sqlalchemy_slave.url"])
```

Comment:
I recommend adding `pool_recycle=30` to every SQL engine. Now a day, idle connections are closed by the server after 120s.

Reply:
If you don't do that, you'll have errors!

Reply:
done :-)